.env

```diff
@@ -7,3 +7,5 @@ export COPY_SSH=1
 # Default EBcL APT repository.
 export EBCL_REPO_URL="http://linux.elektrobit.com/eb-corbos-linux"
 export EBCL_VERSION="1.4"
+# Skip clean for robot tests
+export SDK_ROBOT_SKIP_CLEAN="1"
```

Comment:
@wintron04 @antonjahn I would skip the clean rebuild locally by default, and do a clean rebuild in the CI by default.